### PR TITLE
feat-made dismiss and badge card

### PR DESCRIPTION
**Pull request by therajatg**

### CODE REVIEW

**Review by RahulSingh9131 — APPROVED** [review]
Everything is looking good to me, if possible make utility classes for flex , flex-direction and align-items. Then you can use them anywhere you want.

File: docs/card.html [hunk 1]
```diff
@@ -87,7 +89,7 @@ <h3>Card Subtitle</h3>
     
     <div class="content">
         <p>3. Vertical Card:</p>
-        <div class="card-container vertical-card">
+        <div class="card-container">
             <img src="/pics/product/woodland shoe.jpg">
```
Comment (rahulcs754) [inline docs/card.html:93]: your image file name should be like woodland_shoe.jpg
bcoz i think some time server not read this time file name please check and confirm

File: styles/component-styles/card.css [hunk 2]
```diff
@@ -1,55 +1,72 @@
...
+  display: inline-block;
+  position: absolute;
+  z-index: 1;
+  transform: translate(4rem, 10rem);
+}
+
+.card-badge {
+  background-color: var(--primary-color);
+  height: 22px;
+  width: 40px;
```
Comment (rahulcs754) [inline styles/component-styles/card.css:60]: you can set this px to rem

**Review by rahulcs754 — APPROVED** [review]
The code looks good  and clean 
just small suggestion

**Pull request merged**